dotCMS/src/main/java/com/dotmarketing/factories/TreeFactory.java

```diff
@@ -23,61 +22,32 @@
  */
 public class TreeFactory {
 
-	private static final String CHILD_RELATION_TYPE = "child";
 
-	public static Tree getTree(String x) {
-		try {
-			return (Tree) new HibernateUtil(Tree.class).load(Long.parseLong(x));
-		} catch (Exception e) {
-			Tree tree = null ;
-			try {
-				tree =  (Tree) new HibernateUtil(Tree.class).load(x);
-			} catch (DotHibernateException e1) {
-				Logger.error(TreeFactory.class, "getTree failed:" + e, e);
-			}
-			return tree;
-		}
-	}
 
-	public static Tree getTree(Tree object) {
-		try {
-			return (Tree) new HibernateUtil(Tree.class).load(Tree.class, object);
-		} catch (Exception e) {
-			return new Tree();
-		}
-	}
-	
-	public static Tree getTree(Inode parent, Inode child) {
-		return getTree(parent, child, CHILD_RELATION_TYPE);
+
+	public static Tree getTree(final Inode parent,final  Inode child) {
+
+		return getTree(parent.getInode(), child.getInode());
 	}
 
-	public static Tree getTree(String parent, String child) {
-		return getTree(parent, child, CHILD_RELATION_TYPE);
+	public static Tree getTree(final String parent, final String child) {
+		String relationType = "child";
+		return getTree(parent, child, relationType);
+
 	}
 
-	public static Tree getTree(String parent, String child, String relationType) {
+	public static Tree getTree(final String parent, final String child, final String relationType) {
 		try {
-			final DotConnect dc = new DotConnect();
+			DotConnect dc = new DotConnect();
 			String query = "select * from tree where parent = ? and child = ? ";
 			if(relationType != null) query += " and relation_type = ?";
-
 			dc.setSQL(query);
 			dc.addParam(parent);
 			dc.addParam(child);
-			if(relationType != null) {
-				dc.addParam(relationType);
-			}
-
-			final List<Map<String, Object>> res = dc.loadObjectResults();
-			if(res!=null && !res.isEmpty()) {
-				final Tree tree  = new Tree();
-				final Map resultMap = res.get(0);
-				tree.setChild(resultMap.get("child").toString());
-				tree.setParent(resultMap.get("parent").toString());
-				tree.setRelationType(resultMap.get("relation_type").toString());
-				tree.setTreeOrder(Integer.parseInt(resultMap.get("tree_order").toString()));
-				return tree;
-			}
+			if(relationType != null) dc.addParam(relationType);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Missing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00121)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00122)